components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/storage/model/CredentialsRequestStrategy.java

```diff
@@ -0,0 +1,13 @@
+package com.sap.cloud.cmp.ord.service.storage.model;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class CredentialsRequestStrategy {
+    @Column(name = "type", length = Integer.MAX_VALUE)
+    private String type;
```

Comment:
in ORD this credentials request strategy has an attached description that points to a URL with strategy specification. Probably it makes sense for this to be discoverable here as well

Reply:
In ORD spec CredentialsRequestStrategy has only these two fields, probably we forgot it in the spec?